definitions/artifacts/azure-storage-account-blob.json

```diff
@@ -2,10 +2,10 @@
   "$schema": "http://json-schema.org/draft-07/schema",
   "$md": {
     "access": "public",
-    "name": "azure-storage-account"
+    "name": "azure-storage-account-blob"
```

Comment:
This won’t unpublished the old artifact definition will need to make sure that someone deletes that.

Reply:
Where all does this need to be deleted? I can take care of updated the connections list in `mass app new` and `mass bundle new`, anywhere else?

Reply:
A record exists in the database for each artifact definition. We'll need to remove it from there.